source/includes/gcp/_routers.md

```diff
@@ -126,3 +126,122 @@ Retrieve a cloud router in a given [environment](#administration-environments).
+| `interfaces`<br/>_Array_                     | An array of interfaces to be added to the selected router.                                                                                                                        |
+| `interfaces[*].name`<br/>_string_            | The name of the interface on the router.                                                                                                                                          |
+| `interfaces[*].ipRange`<br/>_string_         | IP address and range of the interface. The IP range must be in the RFC3927 link-local IP address space. The value must be a CIDR-formatted string, _for example: 169.254.0.1/30_. |
+| `interfaces[*].linkedVpnTunnel`<br/>_string_ | URI of the linked VPN tunnel, which must be in the same region as the router.                                                                                                     |
+
+<!-------------------- ADD BGP PEERS -------------------->
+
+#### Add BGP peer(s)
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/routers/5333546534174463697?operation=addbgppeer"
+```
+
+> Request body example:
+
+```json
+{
+  "bgpPeers": [
+    {
+      "name": "juniper-srx-bgp-5th-p2",
+      "interfaceName": "if-juniper-srx-bgp-3rd-exp",
+      "ipAddress": "169.254.13.1",
+      "peerIpAddress": "169.254.13.2",
+      "peerAsn": 64999
+    }
+  ]
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/routers/:id?operation=addbgppeer</code>
+
+Add one or more BGP peers associated to existing router interface(s)
+
+| Required                                 | &nbsp;                                                                                       |
+| ---------------------------------------- | -------------------------------------------------------------------------------------------- |
+| `bgpPeers`<br/>_Array_                   | An array of BGP peers to be added to the selected router.                                    |
+| `bgpPeers[*].name`<br/>_string_          | The name of the BGP peer association.                                                        |
+| `bgpPeers[*].interfaceName`<br/>_string_ | The router interface through which this BGP peering will be realized.                        |
+| `bgpPeers[*].ipAddress`<br/>_string_     | The router's IP address on the above interface for this BGP peering. Only IPv4 is supported. |
+| `bgpPeers[*].peerIpAddress`<br/>_string_ | The peer IP address to which this BGP peering tries to connect to. Only IPv4 is supported.   |
```

Comment:
There are two "to"s. It can be `The peer IP address to which this BGP peering tries to connect.` or `The peer IP address this BGP peering tries to connect to.`

Reply:
Rephrased in https://github.com/cloudops/cloudmc-api-docs/pull/155/commits/e85e04a860ab69deb275abff2dacc6d47d56eeb0